### PR TITLE
Startup

### PR DESCRIPTION
**Pull request by domo2192**
We have the ability to send emails between users. 

Angel and I are pretty uncertain of where the next step is and would love some input 

Testing is very poor at this point and we are going to continue working on that including adding webmock vcr

### CODE REVIEW

**Review by amsmyth1 — APPROVED** [review]
Looks awesome. Tomorrow we should do a knowledge transfer as we all debugged and worked through VCR/web mock today. So we can help you with that and yall can explain this awesome app!

File: coverage/.resultset.json [hunk 1]
```diff
@@ -1,7 +1,52 @@
 {
   "RSpec": {
     "coverage": {
+      "/Users/dominicpadula/turing/Mod3/projects/yardsourcing/yardsourcing-sendgrid/spec/sendgrid_api_spec.rb": {
```
Comment (dev-named-jordan) [inline coverage/.resultset.json:4]: It appears we need to add coverage to the git ignore file here?

**Review by dev-named-jordan — CHANGES_REQUESTED** [review]
One change to look at with regard for coverage in git ignore please =)

File: coverage/.resultset.json (hunk 1 above)
Comment (domo2192) [reply coverage/.resultset.json:4]: Ya good call on that. We can fix that asap

Comment (abreaux26) [reply coverage/.resultset.json:4]: @jordanfbeck0528 we have coverage marked in our .gitignore file, so I am not sure why that showed up.. 🤷🏽‍♀️

**Comment by DougWelchons** [comment]
Angel and I got webmock and VCR working with the Sendgrid API.

File: coverage/.resultset.json (hunk 1 above)
Comment (DougWelchons) [reply coverage/.resultset.json:4]: I double checked that coverage is in the git ignore in my last commit. should be good to go!

Comment (dev-named-jordan) [reply coverage/.resultset.json:4]: sounds good

**Review by dev-named-jordan — APPROVED** [review]

File: spec/sendgrid_api_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+RSpec.describe SandgridApi do
+include Rack::Test::Methods
+
+  def app
+    SandgridApi.new
+  end
+
+  it 'can send emails' do
+    VCR.use_cassette('send_message') do
+      post '/mail?to=doug.welchons@gmail.com&from=angelbreaux@hotmail.com&booking_name=BDAY BASH&description=You are Invited to my party!'
```
Comment (jbranham1) [inline spec/sendgrid_api_spec.rb:16]: I hope I get this invite

Comment (Gvieve) [reply spec/sendgrid_api_spec.rb:16]: I need one too!

**Review by jbranham1 — APPROVED** [review]
Looks fantastic from what I can tell!

**Pull request merged**